clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -21,6 +21,20 @@
    :type       type-name
    :attributes (dissoc value :id)})
 
+(defn- update-vals [map vals f]
+  (reduce #(update-in %1 [%2] f) map vals))
+
+(defn- parse-date [value]
```

Comment:
What about straight-up Clojure namespaces. Or is that just encapsulation by obscurity, if you know what I mean?